checkov/terraform/module_loading/proxy_client.py

```diff
@@ -0,0 +1,36 @@
+import os
+from typing import Any
+
+import requests
+from requests import Request
+
+PROXY_DOME_HEADER = 'X-Proxydome-Identity'
+
+
+class ProxyClient:
+    def __init__(self) -> None:
+        self.proxy_ca_path = os.getenv('PROXY_CA_PATH', None)
+        if self.proxy_ca_path is None:
+            raise Exception("[ProxyClient] CA certificate path is missing")
+
+    def get_session(self) -> requests.Session:
+        if not os.getenv('PROXY_URL', None):
+            raise Exception("Proxy dome is disabled")
```

Comment:
can you change the message to be more general (like `Please provide "PROXY_URL" env var`)